src/main/proto/wfa/measurement/api/v2alpha/exchange_step.proto

```diff
@@ -36,10 +36,20 @@ message ExchangeStep {
   // Output-only.
   State state = 2;
 
-  // Output-only. Denotes the step of the grandparent RecurringExchange's
-  // ExchangeWorkflow that this ExchangeStep corresponds to.
+  // Denotes the step of the grandparent RecurringExchange's ExchangeWorkflow
+  // that this ExchangeStep corresponds to. Output-only.
   ExchangeWorkflow.Step step = 3;
 
+  // Output-only. Once predecessor steps complete, this will contain an entry
+  // for each `step.input_labels` entry produced by the other party. The keys
+  // here are the values of that map.
```

Comment:
by `values of that map`, do we mean `labels`?

Reply:
Clarified the description -- let me know if this is better.